.pre-commit-config.yaml

```diff
@@ -65,7 +65,7 @@ repos:
           - --comment-style
           - "|#|"
   - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
We may as well change this to the new URL:
`- repo: https://github.com/astral-sh/ruff-pre-commit`

Reply:
Also, v0.3.5 is out now.

Reply:
Updated!